examples/NumCosmo/Example2_Fit_Halo_Mass_to_Shear_Catalog_NC.ipynb

```diff
@@ -0,0 +1,944 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Fit halo mass to shear profile: 2. realistic data\n",
+    "\n",
+    "_the LSST-DESC CLMM team_\n",
+    "\n",
+    "\n",
+    "This notebook demonstrates how to use `clmm` to estimate a WL halo mass from observations of a galaxy cluster. It uses several functionalities of the support `mock_data` module to produce datasets of increasing complexity. This notebook also demonstrates the bias introduced on the reconstructed mass by a naive fit, when the redshift distribution of the background galaxies is not properly accounted for in the model. Organization of this notebook goes as follows:\n",
+    "\n",
+    "- Setting things up, with the proper imports.\n",
+    "- Generating 3 datasets: an ideal dataset (dataset1) similar to that of Example1 (single source plane); an ideal dataset but with source galaxies following the Chang et al. (2013) redshift distribution (dataset2); a noisy dataset where photoz errors and shape noise are also included (dataset3). \n",
+    "- Computing the binned reduced tangential shear profile, for the 3 datasets, using logarithmic binning.\n",
+    "- Setting up the \"single source plane\" model to be fitted to the 3 datasets. Only dataset1 has a single source plane, so we expect to see a bias in the reconstructed mass when using this model on datasets 2 and 3. \n",
+    "- Perform a simple fit using NumCosmo tools to compute the best-fit and the Fisher Matrix, and visualize the results.\n",
+    "- Perform a MCMC analysis using NumCosmo tools and visualize the results.\n",
+    "\n",
+    "Note that this notebook is equivalent to `Example2_Fit_Halo_Mass_to_Shear_Catalog.ipynb` except for the statistical analysis, where here we use some NumCosmo tools and we fit both the cluster mass and the concentration parameter."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Setup"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "First, we import some standard packages."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# For NumCosmo\n",
+    "import os\n",
+    "import sys\n",
+    "import gi\n",
+    "\n",
+    "gi.require_version('NumCosmo', '1.0')\n",
+    "gi.require_version('NumCosmoMath', '1.0')\n",
+    "from gi.repository import GObject\n",
+    "from gi.repository import NumCosmo as Nc\n",
+    "from gi.repository import NumCosmoMath as Ncm\n",
+    "\n",
+    "from scipy.stats import chi2\n",
+    "import corner\n",
```

Comment:
This package is not installed in our default instructions, maybe add a comment here to tell the user install it if he does not have it

Reply:
Added a comment in both the introduction text and just above the import.